### PR TITLE
dxLookup - Add itemCenteringEnabled option for the Material theme

### PR DESCRIPTION
**Pull request by groshenkovamarina**

### CODE REVIEW

File: js/ui/lookup.js [hunk 1]
```diff
@@ -233,6 +233,14 @@ const Lookup = DropDownList.inherit({
             * @hidden
             */
 
+            /**
+             * @name dxLookupOptions.disableOptionCentering
+             * @type boolean
+             * @default false @for Material
+             * @public
+             */
+            disableOptionCentering: true,
```
Comment (babich-a) [inline js/ui/lookup.js]: Was this option name discussed with technical writers.

File: js/ui/lookup.js [hunk 2]
```diff
@@ -303,11 +311,9 @@ const Lookup = DropDownList.inherit({
 
                     showPopupTitle: false,
 
-                    position: {
```
Comment (babich-a) [inline js/ui/lookup.js:306]: Should we change the description of [position](https://js.devexpress.com/Documentation/20_1/ApiReference/UI_Widgets/dxLookup/Configuration/#position) for Material theme

File: js/ui/lookup.js [hunk 3]
```diff
@@ -294,7 +302,7 @@ const Lookup = DropDownList.inherit({
 
                     closeOnOutsideClick: true,
 
-                    popupWidth: (function() { return $(this.element()).outerWidth(); }).bind(this),
+                    popupWidth: (function() { return this._getPopupWidth(); }).bind(this),
```
Comment (babich-a) [inline js/ui/lookup.js:305]: It seems we should change the description of [popupWidth](https://js.devexpress.com/Documentation/20_1/ApiReference/UI_Widgets/dxLookup/Configuration/#popupWidth) option

File: js/ui/lookup.js (hunk 1 above)
Comment (DokaRus) [inline js/ui/lookup.js]: Has it been discussed with @RomanTsukanov  or @AlexJameson ?
Almost all of our component options are permissive, not prohibitive (`allowSomething`, `featureEnabled` etc)

File: js/ui/lookup.js [hunk 4]
```diff
@@ -509,11 +518,23 @@ const Lookup = DropDownList.inherit({
...
+            } else {
+                this.callBase();
+            }
         } else {
-            this.callBase();
+            if(this.option('usePopover') && this.option('disableOptionCentering')) {
+                this._renderPopover();
+            } else {
+                this.callBase();
+            }
```
Comment (DokaRus) [inline js/ui/lookup.js]: Can you refactor it?

File: js/ui/lookup.js [hunk 5]
```diff
@@ -963,6 +994,10 @@ const Lookup = DropDownList.inherit({
                 break;
             case 'cleanSearchOnOpening':
             case '_scrollToSelectedItemEnabled':
+            case 'disableOptionCentering':
+                if(this.option('_scrollToSelectedItemEnabled') && !value) {
```
Comment (DokaRus) [inline js/ui/lookup.js]: Is it OK that `cleanSearchOnOpening` and `_scrollToSelectedItemEnabled` will be handled in the same way as `disableOptionCentering`

**Review by babich-a — CHANGES_REQUESTED** [review]

File: js/ui/lookup.js [hunk 6]
```diff
@@ -963,6 +994,10 @@ const Lookup = DropDownList.inherit({
                 break;
             case 'cleanSearchOnOpening':
             case '_scrollToSelectedItemEnabled':
```
Comment (babich-a) [inline js/ui/lookup.js:990]: It seems that we already have the option that enabled an item centering  in material theme. Can we use only one option for that purposes.

File: js/ui/lookup.js (hunk 1 above)
Comment (RomanTsukanov) [reply js/ui/lookup.js]: You've got a point. Maybe `itemCenteringEnabled`?

**Review by babich-a — APPROVED** [review]

**Review by RomanTsukanov — APPROVED** [review]

File: js/ui/lookup.js (hunk 5 above)
Comment (groshenkovamarina) [reply js/ui/lookup.js]: Fixed

File: js/ui/lookup.js (hunk 4 above)
Comment (groshenkovamarina) [reply js/ui/lookup.js]: Done

File: js/ui/lookup.js (hunk 1 above)
Comment (groshenkovamarina) [reply js/ui/lookup.js]: Done

Comment (groshenkovamarina) [reply js/ui/lookup.js]: The option name is 'itemCenteringEnabled' now

File: js/ui/lookup.js (hunk 2 above)
Comment (groshenkovamarina) [reply js/ui/lookup.js:306]: A technical writer will fix after merge

File: js/ui/lookup.js (hunk 3 above)
Comment (groshenkovamarina) [reply js/ui/lookup.js:305]: A technical writer will fix after merge

**Pull request merged**